tulip/transys/labeled_graphs.py

```diff
@@ -1117,7 +1117,11 @@ def remove_deadends(self):
                  if d == 0}
             self.states.remove_from(s)
         m = len(self)
-        assert n == 0 or m > 0, 'removed all {n} nodes!'.format(n=n)
+        assert n == 0 or m > 0, ( 
+            'removed all {n} nodes!'.format(n=n) + '\n'
+            ' Please check env_init and env_safety to avoid trivial'
+            ' realizability. Alternatively, you can set "rm_deadends = 0"'
```

Comment:
In the documentation of `synthesize()`, we write that the type of `rm_deadends` is `bool`, whereas this message suggests using 0. At runtime this is not important because `rm_deadends` is used in a branch condition like `if rm_deadends`, so the consequence is the same whether 0 or `False`.